src/test/regress/input/default_tablespace.source

```diff
@@ -76,4 +76,12 @@ drop database database_for_default_tablespace;
 drop database database_with_tablespace;
 
 drop tablespace some_default_tablespace;
-drop tablespace some_database_tablespace;
\ No newline at end of file
+drop tablespace some_database_tablespace;
+
+-- ensure modified GUCs when starting a new slice does not error
+create tablespace some_default_tablespace location '@testtablespace@_default_tablespace';
+set default_tablespace to some_default_tablespace;
+drop tablespace some_default_tablespace;
+create table table_under_database_with_default_tablespace (a int);
+insert into table_under_database_with_default_tablespace select random() from generate_series(1, 10);
```

Comment:
why we need `random()` here? Can't we just insert sequential values from `generate_series()`?

Reply:
We need `random` here so that ORCA will build a plan with two slices, like a postgres planner, otherwise the error will not appear on ORCA.